flink-sequence-features/data/expected_output.csv

```diff
@@ -0,0 +1,4 @@
+,user_id,click_history
```

Comment:
Can we make the expected output more readable by removing the index column?

This is also needed for consistency with other examples.